### ISSUE TITLE
java.lang.NoClassDefFoundError: org/codehaus/jackson/map/ObjectMapper

### ISSUE THREAD

**Issue by moujan** [chinoio/chino-java#1]
Hi,

I am trying to use the JAVA SDK for Chino and when I try to create a simple ChinoAPI object, I am getting an error, which says that ObjectMapper class is not found:

java.lang.NoClassDefFoundError: org/codehaus/jackson/map/ObjectMapper
	at io.chino.client.ChinoBaseAPI.initializeMapper(ChinoBaseAPI.java:93)
	at io.chino.client.ChinoBaseAPI.<init>(ChinoBaseAPI.java:27)
	at io.chino.client.Applications.<init>(Applications.java:17)
	at io.chino.client.ChinoAPI.initializeObjects(ChinoAPI.java:54)
	at io.chino.client.ChinoAPI.<init>(ChinoAPI.java:29)

I generate the jar for Chino derived with maven pom.xml. The dependencies seem to be fine. 

I cannot find where the problem is! Maybe because my knowledge in maven is limitted! 

I am using jHipster built with maven.

Can you help me? Thank you!

**Comment by zidia** [comment]
Hi Moujan,
did you use the "release" jar or did you try to generate the jar from the code? 

- in case you used our released jar, please try to use the code since it wasn't updated.
- if you were using the code there was a small issue in the project configuration that I fixed today.

Let me know if it works. 

I tried to create a simle Java project adding chino maven dependency
        `<dependency>
	  	<groupId>io.chino.library</groupId>
	  	<artifactId>chino-java-library</artifactId>
		<version>0.0.1-SNAPSHOT</version>
	</dependency>`
and it worked without errors.

cheers
jovan

**Comment by moujan** [comment]
Hi Zidia,
Thank you for the response!

I fixed it. But I am not sure whether it is the solution. I had to add the dependencies for artifacts **jackson-core-asl** and **jackson-mapper-asl**.
In the maven repository site it says that these repositories have been moved into com.fasterxml.jackson.core which is already used in the JDK.

Anyways, thank you very much for the help! I'll try your changes then!

regards,
Loukas
